assets/locales/en/translation.json

```diff
@@ -752,6 +752,23 @@
 		"title": "Batch (<span class=\"text-success\">New</span>)",
 		"description": "<span className=\"bold\">Batch Simming</span> is a new feature akin to the <span className=\"bold\">Top Gear</span> sim on <a href=\"https://raidbots.com\" target=\"_blank\">Raidbots.com</a> that allows you to select multiple items and sim them find the best combinations.<br />This is an <span className=\"text-brand\">Alpha</span> feature, so if you have feedback or find a bug, please report it!",
 		"download_local": "Download the local sim for faster results.",
+		"slots": {
```

Comment:
This should be moved to `character.json` translations as this can be used across the whole application, ideally you also move `gear_tab.slots`.
We should have:
- `itemSlot` > This can be Finger 1, Finger 2 etc
- `itemType` > This will be Finger, Trinket
Just like we have in the proto

Reply:
Updated with bunch of other stuff !